factory/post.js

```diff
@@ -26,7 +25,10 @@ function processContent(data) {
     var item = entries[i],
         postTitle = item.title.$t,
         content   = item.content.$t,
-        url       = item.link[4].href,
+        // Do not assume the link will always remain item 4 in the array
+        url       = _.filter(item.link, function (link) {
+          return link.rel === 'alternate' && link.type === 'text/html';
+        })[0].href,
```

Comment:
Still trying to wrap my head around what you did here. Won't the link rel and type always be `text/html` and `alternate`?  So why are we checking that? and how does it solve the assumption?